backend/composer/api/urls.py

```diff
@@ -27,6 +28,11 @@
     ConnectivityStatementViewSet,
     basename="connectivity-statement",
 )
+router.register(
```

Comment:
question: @D-GopalKrishna what is the difference between Generic and "normal" connectivity statements?